SPECS/openssh/openssh.spec

```diff
@@ -2,24 +2,21 @@
 
 %global pam_ssh_agent_ver 0.10.3
 
-%define systemd_units_rel 20191026
-
 Summary:        Free version of the SSH connectivity tools
 Name:           openssh
 Version:        %{openssh_ver}
-Release:        4%{?dist}
+Release:        5%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Security
 URL:            https://www.openssh.com/
 Source0:        https://ftp.usa.openbsd.org/pub/OpenBSD/OpenSSH/portable/%{name}-%{openssh_ver}.tar.gz
-Source1:        http://www.linuxfromscratch.org/blfs/downloads/stable-systemd/blfs-systemd-units-%{systemd_units_rel}.tar.xz
```

Comment:
Please update the signatures file to account for the removal of the old `Source1` file.

Reply:
Yes, can do - I carefully considered if I'd _added_ any new sources, but completely forgot that I'd removed one 😀